### PR TITLE
[zh] Update /releases/_index.md

### PR DESCRIPTION
**Pull request by yanrongshi**

### CODE REVIEW

**Comment by yanrongshi** [comment]
@howieyuen

**Comment by Sea-n** [comment]
You have opened four PR for very small (and un-noticeable for current version) sync. Could you merge them into one?
- #34406
- #34407
- #34408
- #34409

Futher more, it's better to fix format errors and make sentences more fluent.

**Comment by howieyuen** [comment]
see the [deploy log](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62af6da775616800098c0c5f#L76):
```
2:41:36 AM: Error: Error building site: "/opt/build/repo/content/zh-cn/releases/_index.md:16:122": failed to extract shortcode: template for shortcode "currentVersionAddMinor" not found
```

**Comment by howieyuen** [comment]
please squash commits into one
/label tide/merge-method-squash

File: content/zh-cn/releases/_index.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ type: docs
 <!-- overview -->
 
 <!-- 
-The Kubernetes project maintains release branches for the most recent three minor releases ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}).  Kubernetes 1.19 and newer receive [approximately 1 year of patch support](/releases/patch-releases/#support-period). Kubernetes 1.18 and older received approximately 9 months of patch support.
+The Kubernetes project maintains release branches for the most recent three minor releases ({{< skew currentVersion >}}, {{< skew currentVersionAddMinor -1 >}}, {{< skew currentVersionAddMinor -2 >}}).  Kubernetes 1.19 and newer receive [approximately 1 year of patch support](/releases/patch-releases/#support-period). Kubernetes 1.18 and older received approximately 9 months of patch support.
 -->
-Kubernetes 项目维护最近三个次要版本（{{< skew latestVersion >}}、{{< skew prevMinorVersion >}}、{{< skew oldMinorVersion >}}）的发布分支。
+Kubernetes 项目维护最近三个次要版本（{{< skew currentVersion >}}、{{< currentVersionAddMinor -1 >}}、{{< currentVersionAddMinor -2 >}}）的发布分支。
```
Comment (howieyuen) [inline content/zh-cn/releases/_index.md:18]:
```suggestion
Kubernetes 项目维护最近三个次要版本（{{< skew currentVersion >}}、{{< skew currentVersionAddMinor -1 >}}、{{< skew currentVersionAddMinor -2 >}}）的发布分支。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Pull request merged**